scripts/pylint_extensions.py

```diff
@@ -162,10 +178,12 @@ def process_module(self, node):
         Args:
             node: astroid.scoped_nodes.Function. Node to access module content.
         """
-        file_content = node.stream().readlines()
+        file_content = [
+            utils.convert_to_unicode(line) for line in
+            node.stream().readlines()]
         file_length = len(file_content)
         exclude = False
-        for line_num in xrange(file_length):
+        for line_num in range(file_length):
```

Comment:
Should we put this in `python_utils` to keep the benefits of having `xrange` be a generator?

Reply:
Actually this is Python3's range, i.e it already is a generator and not a list. Its imported from the buitlins module:
`from builtins import range`